lib/rubocop/cop/thread_safety/mutable_class_instance_variable.rb

```diff
@@ -0,0 +1,227 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module ThreadSafety
+      # This cop checks whether some class instance variable isn't a
+      # mutable literal (e.g. array or hash).
+      #
+      # It is based on Style/MutableConstant from RuboCop.
+      # See https://github.com/rubocop-hq/rubocop/blob/master/LICENSE.txt
```

Comment:
I'm surprised to see a link to the license. Are we trying to protect from a copyright complaint? I would imagine a link to the original implementation would be helpful to a developer trying to follow where this code came from.

Reply:
I was following the pattern used in https://github.com/rubocop-hq/rubocop-extension-generator/blob/master/lib/rubocop/extension/generator/generator.rb#L30

It does seem excessive to link to the license though, as long as a link to the original code is present. Part of my reasoning for mentioning the MutableConstant cop is that it would be a source for potential updates later if the maintainers of rubocop make changes to patterns detected as immutable.

I'll change it up to a link to the code instead.